### PR TITLE
Created video solution for gold problems (usaco-647, usaco-1091, usaco-694, usaco-898)

### PR DESCRIPTION
**Pull request by cskitty**

### CODE REVIEW

File: solutions/usaco-647.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+id: usaco-647
+source: USACO Gold 2016 Open
+title: 248
```
Comment (maggie-j-liu) [inline solutions/usaco-647.mdx:4]: the title is being parsed as a number, not sure if that's a problem? @thecodingwizard

File: solutions/usaco-1091.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+id: usaco-1091
```
Comment (maggie-j-liu) [inline solutions/usaco-1091.mdx:2]: > Couldn't find corresponding problem for internal solution with frontmatter ID usaco-1091. If this problem is no longer in any module, add it to content/extraProblems.json.

*https://vercel.com/cpinitiative/usaco-guide/CLkHRJJStkKpg34HZW6eKF4UzaZM*

I think this is breaking the build

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/usaco-647.mdx (hunk 1 above)
Comment (thecodingwizard) [reply solutions/usaco-647.mdx:4]: might be, idk. Just wrap it with quotes
```suggestion
title: '248'
```

**Review by SansPapyrus683 — APPROVED** [review]
wait why hasn't this been merged alr

**Review by dongliu0426 — APPROVED** [review]
nice vids

**Pull request merged**